include/seqan3/range/view/all.hpp

```diff
@@ -114,48 +114,97 @@
  * // vec_view4 and vec_view5 are the reverse complement of "ACGGTC": "GACCGT"
  * ```
  *
- * \par View properties
+ * ### Views vs view adaptors
  *
- * All of these are documented for SeqAn3's views individually, but only explained here in detail:
+ * When talking about views, two different entities are often conflated:
  *
- * **Source-only views:** Most views operate on an input range and return a (modified) range, i.e. they can be placed
+ *   1. the view (this is the type that is a range and meets seqan3::view_concept; it is what we refer to with
+ * `auto vec_view` above)
+ *   2. the view adaptor (this is the functor that returns the actual view based on it's parameters, including the
+ * underlying range; in the above example `ranges::view::reverse` and `view::complement` are view adaptors)
+ *
+ * The view adaptor also facilitates the piping behaviour. It is the only entity that is publicly documented and
+ * the actual view type (the range type returned by the adaptor) is considered implementation defined.
+ * The *properties* of the returned range are however specified and documented as part of the adaptor, see below.
+ *
+ * <sub>
+ * An exception to this rule are views that don't work on an underlying range and can only be
+ * placed at the beginning of a pipe of operations; they do not need an adaptor, because their constructor is
+ * sufficient. This is not relevant for the documention, though, we always document `view::foo`, independent of
+ * whether `view::foo` is the adaptor that returns the "foo type" or whether `view::foo` is the "foo type".
+ * </sub>
+ *
+ * ### View properties
```

Comment:
Capitilization

Reply:
We have never discussed this, and while I am open to change, I would rather maintain consistency now and create a ticket for someone to  change it throughout the entire documentation.

→ #244